database/spanner/examples/migrations2/1481574547_create_users_table.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE Users;
```

Comment:
please add empty line at the end (also on other files).

Reply:
Fixed